### DISCUSSION TITLE
Trying to connect through wallet connect to a dapp on Linea

### DISCUSSION THREAD

**Discussion by Julink-eth** [rainbow-me/rainbowkit#2182] [Q&A] [answered]
Hello,

I'm trying to connect to dapps on Linea such as LayerBank or Lynex. 
I use wallet connect QR code on Rainbow wallet but I get the error:
"Connection Failed, There was an issue with the namespaces requested by the dapp. Please try again or contact rainbow and/or dapp support team.
No accounts found in approved namespace." 

Is Linea not supported by Rainbow wallet ?

Thank you

**Comment by DanielSinclair** [comment, marked as answer]
Rainbow Wallet on mobile doesn't support Linea, but other wallets do, and that is supported by RainbowKit. You can also use the Rainbow extension on desktop and add Linea as a Custom Network.

**Reply by Julink-eth** [reply]
Thank you for confirming.
